pdr_backend/aimodel/dash_plots/util.py

```diff
@@ -0,0 +1,60 @@
+import os
+
+import pandas as pd
+
+
+def read_files_from_directory(directory):
+    file_data = {}
+    for filename in os.listdir(directory):
+        if not (filename.endswith(".csv") or filename.endswith(".parquet")):
```

Comment:
I suggest losing the parantheses: if not filename.endswith(".csv") and not filename.endswith(".parquet"). Much easier to read.